Tools/Pipeline/Program.cs

```diff
@@ -45,11 +45,18 @@ static void Main(string [] args)
 			win.Show (); 
 			var model = new PipelineProject();
 			new PipelineController(win, model);  
+			#if LINUX
```

Comment:
Yeah...  i'm wrong.   Github cut off line 41 here which clearly shows this is all within a `#if LINUX || MONOMAC` block.